tools/gluten-te/centos/defaults.conf

```diff
@@ -21,17 +21,28 @@ DEFAULT_DEBUG_BUILD=OFF
 DEFAULT_BUILD_BACKEND_TYPE=velox
 
 # HTTP proxy
-DEFAULT_HTTP_PROXY_HOST=
-DEFAULT_HTTP_PROXY_PORT=
+# If http proxy is http://child-prc.intel.com:913, write it as
+# DEFAULT_HTTP_PROXY_HOST=child-prc.intel.com
```

Comment:
Since `HTTP_PROXY_HOST & HTTP_PROXY_PORT` are already defined at the beginning of velox_be.yml, it seems there is no need to set the default here. Ditto for ubuntu/defaults.conf.